pyvrp/_Individual.pyi

```diff
@@ -47,6 +47,19 @@ class Route:
         """
         Total waiting duration on this route.
         """
+    def total_duration(self) -> int:
+        """
+        Total duration on this route (travel + service + wait - timewarp).
+        It holds that duration = end time - start time.
```

Comment:
No it is not since it depends on the start time, so there is no unambiguous end time. BTW, I noted the comment is wrong (duration does not exclude timewarp).

Reply:
We could make a function `end_time(Duration start_time)`?

Reply:
@N-Wouda any thoughts?